### ISSUE TITLE
Reviewer Main Navigation Elements

### ISSUE THREAD

**Issue by ricardobaeta** [src-d/code-annotation#193]
We have to rethink the main navigation for **Requester**. For the **Worker** we have a solid navigation, but the Requester still needs a little love. We must have a way to move forward and backwards. The **PREVIOUS** dropdown/button should only enable the user to go backwards, and this raises the need of a navigation element to move forward. We must have a **NEXT** button, creating therefore a consistent navigation for this profile.

**Comment by ricardobaeta** [comment]
@dpordomingo @bzz @smacker @carlosms 

Here's the wireframe for the Reviewer's Main Navigation. The **NEXT** button should convey ```btn btn-info``` to couple with Annotator's Main Navigation **SKIP** button.

<img width="1219" alt="screen shot 2018-03-02 at 14 58 54" src="https://user-images.githubusercontent.com/1746146/36905291-efa906f6-1e2a-11e8-9458-250a93988778.png">

Side-note: The **PREVIOUS** button/dropdown should only enable backwards navigation on the survey.

**Comment by dpordomingo** [comment]
Thanks @ricardobaeta for rewording the issue desc; now it's clear!!!

Let me ask one related question:
Should the worker view follow the same logic? I mean: should the `PREVIOUS` button on the Requester view, list only the previous FilePairs?

**Comment by ricardobaeta** [comment]
Yes @dpordomingo :) Previous is only previous :) We're getting feedback on the need to move forward based on ID, or search, but that's other thing we need to tackle in a separate issue :) And after research report.
